### PR TITLE
[NUI] Add Transition Group

### PR DESCRIPTION
**Pull request by bshsqa**
Signed-off-by: Seungho Baek <sbsh.baek@samsung.com>

### API Changes ###

 - ACR:

### CODE REVIEW

File: src/Tizen.NUI/src/internal/Transition/TransitionGroupItem.cs [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+
+            this.useGroupTimePeriod = useGroupTimePeriod;
+            this.stepTransition = stepTransition;
+            this.useGroupAlphaFunction = useGroupAlphaFunction;
+            transitionView = view;
+
+            transitionItemList = new List<TransitionItemBase>();
+            for (int index = 0; index < transitionList.Count; ++index)
+            {
+                TimePeriod localTimePeriod = new TimePeriod(transitionList[index].GetTimePeriod().DelayMilliseconds, transitionList[index].GetTimePeriod().DurationMilliseconds);
```
Comment (dongsug-song) [inline src/Tizen.NUI/src/internal/Transition/TransitionGroupItem.cs:56]: could you check the warning below?
maybe, this warning can be resolved by adding `localTimePeriod.Dispose();` after completely used.

File: src/Tizen.NUI/src/internal/Transition/TransitionItemBase.cs [hunk 2]
```diff
@@ -23,14 +23,16 @@ namespace Tizen.NUI
 
     internal class TransitionItemBase : BaseHandle
     {
+        private bool appearingTransition = true;
+
         /// <summary>
         /// Creates an initialized TransitionItemBase.<br />
         /// </summary>
-        public TransitionItemBase(View target, bool isAppearing, TimePeriod timePeriod, AlphaFunction alphaFunction) : this(Interop.TransitionItemBase.New(), true)
+        public TransitionItemBase(View target, bool appearingTransition, TimePeriod timePeriod, AlphaFunction alphaFunction) : this(Interop.TransitionItemBase.New(), true)
```
Comment (dongsug-song) [inline src/Tizen.NUI/src/internal/Transition/TransitionItemBase.cs:31]: could you resolve the warning below?

**src/Tizen.NUI/src/internal/Transition/TransitionItemBase.cs#L31
Parameter target of method .ctor is never used. Remove the parameter or use it in the method body.**

File: src/Tizen.NUI/src/public/Transition/TransitionGroup.cs [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+using System;
+using System.ComponentModel;
+using Tizen.NUI.BaseComponents;
+using System.Collections.Generic;
+
+namespace Tizen.NUI
+{
+    /// <summary>
+    /// FadeTransition class is a cluster of properties for the fade transition of a View.
```
Comment (jaehyun0cho) [inline src/Tizen.NUI/src/public/Transition/TransitionGroup.cs:26]: I think that the description of this class is required to be modified.

File: src/Tizen.NUI/src/internal/Transition/TransitionGroupItem.cs (hunk 1 above)
Comment (bshsqa) [reply src/Tizen.NUI/src/internal/Transition/TransitionGroupItem.cs]: Thank you I fixed my code to do not need to call Dispose().

File: src/Tizen.NUI/src/internal/Transition/TransitionItemBase.cs (hunk 2 above)
Comment (bshsqa) [reply src/Tizen.NUI/src/internal/Transition/TransitionItemBase.cs:31]: I added Log to use the target argument.

File: src/Tizen.NUI/src/public/Transition/TransitionGroup.cs (hunk 3 above)
Comment (bshsqa) [reply src/Tizen.NUI/src/public/Transition/TransitionGroup.cs]: I fixed the comment of Transition Group.
Please check this changes again.

**Review by dongsug-song — APPROVED** [review]
looks good to me.

**Pull request merged**